openmicroscopy/openmicroscopy
# Permissions Test Matlab : Whole Suite

Testing Instructions:

1) Filet to test : Permissions_Suite.m .
2) Check : Setup database from : /squig/team/colin/from_petr/
![screen shot 2014-06-19 at 18 05 54](https://cloud.githubusercontent.com/assets/6438384/3330735/07ec21de-f7d4-11e3-9e52-8ed0c504ba89.png)

3) Host the database locally (localhost) and start the database.
extract the binary data onto ~/var/
sudo ln -s "/Users/bramalingam" /Users/petr [Note: Please replace the user path with yours]
createdb -O omero test-permissions-from-dump
pg_restore -h localhost -U omero -Fc -d test-permissions-from-dump ~/annotated.db.dump

4) Now download the latest version of Omero-Matlab toolbox.
5) Add it to the MatlabPath [ type the following : add path(genpath(<path to matlab toolbox>))] in the command window.
6) Run the startup script [Startup.m]. --> sets up the class path.
7) Run the Permissions_Suite.m
